spec/octopus/serverspec/json/apidoesnotsupportspaces.json

```diff
@@ -0,0 +1,120 @@
+{
+  "Application": "Octopus Deploy",
+  "Version": "2018.4.5",
+  "ApiVersion": "3.0.0",
+  "InstallationId": "7868cc59-81b3-4d44-add8-703216791dca",
```

Comment:
MIght wanna tweak that id.

Reply:
I generated a fresh one just for that sample file